pkg/gotohelm/testdata/src/example/bootstrap/_shims.tpl

```diff
@@ -1,34 +1,80 @@
 {{- /* Generated from "" */ -}}
 
+{{- /* Generated from "_shims.go" */ -}}
```

Comment:
Can the first line be deleted to not repeat `Generated from ...`?

Reply:
Yes, I can update the "bundler" to no include that line. Good call!